anthos-bm-gcp-terraform/terraform.tfvars.sample

```diff
@@ -15,5 +15,5 @@ instance_count                = {
 #nfs_server                   = true
 #gpu                          = {
 #                                 count = 1,
-#                                 type = "nvidia-tesla-k80"
+#                                 type = "nvidia-tesla-t4"
```

Comment:
Question: Is there a reason for this change (`nvidia-tesla-k80` to `nvidia-tesla-t4`)?

Reply:
Yes, the `nvidia-tesla-k80` are EOL and no longer available (blocks the CI build): https://cloud.google.com/compute/docs/eol/k80-eol